demo/malloy-demo-composer/src/electron/main.ts

```diff
@@ -126,4 +126,69 @@ async function registerIPC(): Promise<void> {
   ipcMain.handle("post:open_directory", async (_event) => {
     return await getOpenDirectory();
   });
+
+  // Native application menu
+  const template: (Electron.MenuItem | Electron.MenuItemConstructorOptions)[] =
+    [
+      { role: "appMenu" },
+      { role: "fileMenu" },
+      { role: "editMenu" },
+      {
+        label: "View",
+        submenu: [
+          { role: "reload" },
+          { role: "forceReload" },
+          { role: "toggleDevTools" },
+          { type: "separator" },
+          { role: "resetZoom" },
+          { role: "zoomIn" },
+          { role: "zoomOut" },
+          { type: "separator" },
+          { role: "togglefullscreen" },
+          { type: "separator" },
+          {
+            label: "View Open Source Licenses",
+            click: async () => {
+              const thirdPartyWindow = new BrowserWindow({
+                width: 400,
+                height: 600,
+                webPreferences: {
+                  javascript: false,
+                },
+              });
+
```

Comment:
Having the full menu visible on the license popup seems weird since most of the actions are not relevant to the popup. Maybe we should we just hide the menu by adding something like: 
```suggestion
thirdPartyWindow.setMenu(null);
```

Reply:
this took me a minute to understand, til I saw that this menu type is only available on linux/windows, which I guess is why I wasn't seeing it :D